PubSub/src/Subscription.php

```diff
@@ -272,6 +272,23 @@ public function name()
      *           might use "https://example.com/push".
      *     @type bool $retainAckedMessages Indicates whether to retain
      *           acknowledged messages.
+     *     @type array $retryPolicy A policy that specifies how Cloud Pub/Sub
+     *           retries message delivery for this subscription. If not set, the
+     *           default retry policy is applied. This generally means that
+     *           messages will be retried as soon as possible for healthy
+     *           subscribers. Retry Policy will be triggered on NACKs or
+     *           acknowledgement deadline exceeded events for a given message.
+     *           Retry Policies are implemented on a best effort basis At times,
+     *           the delay between deliveries may not match the configuration.
+     *           That is, the delay can be more or less than the configured
+     *           backoff. **Please note** that this feature may not be
```

Comment:
Is this usual that features are not available on Rest?

Reply:
Generally pre-GA features don't appear in the json schema discovery document. Since the rest transport relies on that, it's not supported yet.

Reply:
I see. The wording confused me. Can we make it clear that only create/update calls with the feature would not honor the feature? If you created the subscription with retry policy already, other requests like pull will still honor the retry policy.

Reply:
Ping.

Reply:
Looks like the feature was published in the [service definition](https://github.com/googleapis/discovery-artifact-manager/blob/c0b33783d93f60b93fa730ff7ca1bfd25c5d879b/discoveries/pubsub.v1.json#L209-L212), so I'm removing the notice altogether.